README.md

```diff
@@ -25,29 +23,41 @@ strategy = {
 };
 ```
 
-`matchObject` MUST be a RegExp, a String or an Array of String. When it is a RegExp, it SHOULD include 'g' flag.
+`matchObject` MUST be a RegExp, a String or an Array of either. When it is a RegExp, it SHOULD include 'g' flag.
 
 ```js
 matchObject = 'abc';  // every 'abc' match
+matchObject = /\B@\w+/g; // every word starting with @ match
+
+// For arrays, without allowMultiPartMatching:
 matchobject = ['a', 'b', 'c'];  // every 'a' 'b' and 'c' match
-matchObject = /\B@\w+/g; // every words start with @ match
+
+// For arrays, with allowMultiPartMatching, each strategy in the array is applied to the results of the prior strategy, in left to right order.
+matchobject = ['/\{.*\}/', 'abc']; // every 'abc' that occurs within '{' and '}' match
```

Comment:
At minimum, I should make this `matchObject` rather than `matchobject`, which is just an error on my part.

`matchesObject` would be more semantic, but the `matchObject` naming comes from upstream and is used pretty widely throughout, so changing makes future conflicts from upstream more likely. Not sure if we care about that at this point, but calling it out anyway.